### DISCUSSION TITLE
Eightree ET38 ESP32-C3 Tasmota 13.4 german

### DISCUSSION THREAD

**Discussion by tom8184** [arendst/Tasmota#20918] [Support request] [answered]
Hello,

I could update Eightree ET38 from Tasmota 12.5 via Wifi/Browser to Tasmota 13.4 (http://ota.tasmota.com/tasmota32/release/tasmota32c3.bin) but not to the german version (http://ota.tasmota.com/tasmota32/release/tasmota32-DE.bin).
Why? No message is shown.
What I have to do to update from now 13.4 english to the 13.4 german version?

Thanks for any help or idea

**Comment by sfromis** [comment, marked as answer]
What you tried to do is to "upgrade" from an ESP32-C3 specific binary to a binary which is specific to the "original" ESP32. These are very different chip architectures, and cannot use same binaries. The Tasmota project does not provide precompiled language-specific binaries for the newer variants, and therefore you need to build yourself if that's what you want.

**Reply by tom8184** [reply]
Hello,
if I understand it right there is no german firmware for a ESP32-C3.
That is not I want to hear but it is ok. I am not able to build my own german version.

Thank you for the quick answer and information.
best regards

**Reply by sfromis** [reply]
You may think that it is hard to do a custom build, but it is not. If you use TasmoCompiler, you can do it from your browser, and select a few options.
https://github.com/benzino77/tasmocompiler#how-to-start-using-tasmocompiler

**Reply by HofeBY** [reply]
@tom8184

It is possible to compile your own german Tasmota version.
read Liesmich-zuerst.txt

Try out https://github.com/HofeBY/Tasmota-Firmware

Maybe existing C3 Version is OK for you. 

Oberfränkische Grüße
HofeBY

**Reply by tom8184** [reply]
Hello #sfromis,
Hello #HofeBY,

many thanks fo your informations, I will read and try the next days, maybe not before the weekend.

kind regards und Grüße
Tom
